cmd/booster-http/run.go

```diff
@@ -171,8 +180,29 @@ var runCmd = &cli.Command{
 			}()
 		}
 
+		ctxx := lcli.ReqContext(cctx)
+		var ctx context.Context
```

Comment:
Nit, but you don't need `ctxx` and you don't need `else statement`
```suggestion
		ctx := lcli.ReqContext(cctx)
```